BaragonUI/app/components/albs/TargetGroupsTable.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+
+import UITable from '../common/table/UITable';
+import Column from '../common/table/Column';
+
+const targetGroupName = (targetGroup) => {
+  const name = targetGroup.targetGroupName;
+  return <Link to={`/albs/target-groups/${name}`}>{name}</Link>;
+};
+
+const TargetGroupsTable = ({targetGroups}) => {
+  return (
+    <div>
+      <UITable
+        data={targetGroups}
```

Comment:
ah, no problem then. Maybe we'l eventually make a more detailed target group page we'll display it there 👍